client/jetpack-onboarding/steps/contact-form.jsx

```diff
@@ -51,15 +52,15 @@ class JetpackOnboardingContactFormStep extends React.PureComponent {
 					title="Contact Form ‹ Jetpack Start"
 				/>
 
+				<JetpackLogo full size={ 45 } />
+
 				<FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />
 
 				<TileGrid>
 					<Tile
 						buttonLabel={ ! hasContactForm ? translate( 'Add a contact form' ) : undefined }
 						description={
-							hasContactForm
-								? translate( 'Your contact form has been created.' )
-								: translate( 'Not sure? You can skip this step and add a contact form later.' )
+							hasContactForm ? translate( 'Your contact form has been created.' ) : undefined
```

Comment:
The old nitpick -- I prefer to avoid _assigning_ `undefined`; I typically merely _check_ for it (or falsiness, as you do in `Tile` here). When assigning, I prefer to use `null`. 😁

Reply:
(I just find that assigning `undefined` is sort of an oxymoron 😉)

Reply:
Makes sense 👍  Updated in https://github.com/Automattic/wp-calypso/pull/22585/commits/dc3e57a633f069963e68c31fdfada3e833af5383.